packages/solutions/app-tools/src/index.ts

```diff
@@ -237,6 +237,10 @@ export const appTools = (
           .option('-d, --debug', i18n.t(localeKeys.command.new.debug), false)
           .option('--dist-tag <tag>', i18n.t(localeKeys.command.new.distTag))
           .option('--registry', i18n.t(localeKeys.command.new.registry))
+          .option(
+            '--no-need-install',
```

Comment:
How about `--no-install` (which is shorter)

Reply:
It is same with @modern-js/create params, I think not use different params

Reply:
get